applications/fishing-map/src/features/datasets/NewDataset.tsx

```diff
@@ -159,16 +225,11 @@ const DatasetFile: React.FC<DatasetFileProps> = ({ onFileLoaded, className = ''
   )
 }
 
-export type DatasetCustomTypes = 'points' | 'lines' | 'geometries'
 export type DatasetMetadata = {
   name: string
   description?: string
   type: DatasetTypes.Context
-  configuration?: {
-    file?: string
-    type?: DatasetCustomTypes
-    format?: 'geojson'
-  }
+  configuration?: DatasetConfiguration
```

Comment:
Thanks for using an interface here. Is this field still optional?

Reply:
Yes, a very basic dataset doesn't need configuration.